examples/land-cover-fastai/deep_lulc_walkthrough.ipynb

```diff
@@ -84,18 +93,17 @@
     "import shutil\n",
     "import numpy as np\n",
     "from os import makedirs, path as op\n",
+    "import datetime\n",
     "\n",
-    "from sentinelhub import BBoxSplitter, CRS, CustomUrlParam\n",
+    "from sentinelhub import BBoxSplitter, CRS, CustomUrlParam, DataSource\n",
     "\n",
-    "from eolearn.core import FeatureType, SaveToDisk, AddFeature, OverwritePermission, LinearWorkflow, EOPatch\n",
-    "from eolearn.io import S2L1CWCSInput\n",
+    "from eolearn.core import FeatureType, SaveTask, AddFeature, OverwritePermission, LinearWorkflow, EOPatch\n",
```

Comment:
There are still some usages of `SaveToDisk` task down below in the notebook.

Reply:
And some other tasks that need updating as well in the `Step 3. Prediction and model inference` part

Reply:
replaced `SaveTask`

couldn't find others to change, you have them at hand?

Reply:
found them ..